db/migrate/20150204103835_add_state_to_defence_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddStateToDefenceRequests < ActiveRecord::Migration
+  def change
+    add_column :defence_requests, :state, :string
```

Comment:
Yeah the simple_state gem always sets  the field to the default specified in the model, can't really the benefits of having a default at the database level.